ARMTemplates/AzFunctionApp-Consumption.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "appName": {
+            "value": "[concat('fnapp', uniqueString(resourceGroup().id))]"
+        },
+        "storageAccountType": {
+            "value": "Standard_LRS"
+        },
+        "location": {
+            "value": "[resourceGroup().location]"
+        },
+        "runtime": {
+            "value": "node"
```

Comment:
Ours is a `dotnet` runtime, not `node`.

Reply:
thanks....i thought i had this fixed but guess not.